<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDeviceDlg</class>
 <widget class="QDialog" name="addDeviceDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#333333;
}
QPushButton{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
}
#titleWidget{
background:#ffffff;
border-top-left-radius:4px;
border-top-right-radius:4px;
}
#contentWidget{
background:#eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}
#addDeviceBtn{
border:1px solid #cccccc;
border-top-left-radius:14px;
border-bottom-left-radius:14px;
border-right:none;
color:#888888;
}
#addDeviceBtn:checked{
color:#222222;
background:#cccccc;
}
#selectedBtn{
border:1px solid #cccccc;
border-top-right-radius:14px;
border-bottom-right-radius:14px;
color:#888888;
}
#selectedBtn:checked{
color:#222222;
background:#cccccc;
}
QLineEdit{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#666666;
background:#ffffff;
border:1px solid #c3caca;
border-radius:2px;
padding-left:10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>添加球机</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QLabel{
color:#666666;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>105</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>85</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addDeviceBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>新增设备</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectedBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>列表选择</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>152</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>152</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="addDevicePage">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>15</number>
            </property>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>IP地址</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QIPLineEdit" name="ipLineEdit">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>15</number>
            </property>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>端口</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="portLineEdit">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>15</number>
            </property>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>用户名</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userLineEdit">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>15</number>
            </property>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>36</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>36</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>密码</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="passwdLineEdit">
              <property name="minimumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>28</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="selectedPage">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="title" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#title{
background:#ffffff;
border:1px solid #cccccc;
border-bottom:none;
}</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <property name="spacing">
              <number>20</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>90</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>IP</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>端口</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="styleSheet">
             <string notr="true">#listWidget{
border:1px solid #cccccc;
}
QListView:item{
border-bottom:1px solid #cccccc;
}
QListView::item:hover{ 
        background-color: rgba(247, 251, 255);
}
QListView::item:selected{ 
        background-color: rgb(206, 227, 253)
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="hintLabel">
        <property name="styleSheet">
         <string notr="true">color:red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#confirmBtn{
background:#ffd500;
border-radius:2px;
color:#333333;
}
#confirmBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#cancelBtn{
background:#ffffff;
border-radius:2px;
color:#333333;
}
#cancelBtn:pressed{
margin:1px;
}</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QIPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>QIPLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
